src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -140,6 +159,12 @@ public function is_on_section( $section = '' ): bool {
 	 * @return array[]
 	 */
 	public function get_fields(): array {
+
+		// Check to see if we're editing an email, first.
+		if ( $this->is_editing_email() ) {
```

Comment:
Let's check if it's on section https://github.com/the-events-calendar/event-tickets/pull/2550/files#diff-b864484972892032b7a96254575015028df2c5156fd149628898c69c2b99e626R129